redis/src/aio/multiplexed_connection.rs

```diff
@@ -419,4 +473,97 @@ impl ConnectionLike for MultiplexedConnection {
     fn get_db(&self) -> i64 {
         self.db
     }
+
+    fn get_push_manager(&self) -> PushManager {
```

Comment:
the push manager is exposed to user, and also being modified internal by the mpxconn. It's unclear when should the user subscribe through the mpxconn and when through push manager.